src/containers/WebComponentLoader.jsx

```diff
@@ -47,13 +40,18 @@ const WebComponentLoader = (props) => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const [projectIdentifier, setProjectIdentifier] = useState(identifier);
+  const localStorageUser = JSON.parse(localStorage.getItem(authKey));
+  const user = useSelector((state) => state.auth.user);
+  const [loadCache, setLoadCache] = useState(!!!user);
+  const [loadRemix, setLoadRemix] = useState(!!user);
```

Comment:
Not quite sure what's going on with all the `!`s but I assume we're saying if there's a user load remix, otherwise load cache

Reply:
- The ! operator is a unary operator that negates a boolean value. If you apply it to a truthy value, it will return false, and if you apply it to a falsy value, it will return true.
- !! is often used to convert a value to its boolean equivalent. It effectively converts any value to a boolean by applying the logical NOT (!) operator twice.
- !!! is typically used for type coercion in JavaScript. It applies a triple negation to a value, which effectively converts the value to its boolean representation and inverts it twice, which can be used to force a value into a boolean context and obtain a true or false result.
thanks to ChatGPT for the explanations 😅 I had to ask twice about `!!!` but it's just ensuring its a boolean